tooling/cli/Cargo.toml

```diff
@@ -42,7 +42,7 @@ path = "src/main.rs"
 openssl-vendored = [ "tauri-mobile/openssl-vendored" ]
 
 [dependencies]
-tauri-mobile = { version = "0.4", default-features = false }
+tauri-mobile = { git = "https://github.com/tauri-apps/tauri-mobile", default-features = false }
```

Comment:
Let's publish tauri-mobile first.

Reply:
yeah sure, this is mainly to pass in CI and will bring it back before merge and after tauri-mobile is released